themes/default/shop/config.json5

```diff
@@ -2,7 +2,7 @@
     assetPrefix: "shop",
     music: "home.backgroundMusic",
     background: {
-        items: [{ key: "home.background", x: 0, y: 0 }],
+        items: [{ key: "shop.magicShop", x: 0, y: 0 }],
```

Comment:
This whole config file is _probably_ orphaned but I've just conformed it to the same as the debug config for now. We might be able to ditch this once upcoming tix around collections and debug mode but at present we need a shop to exist in main.js.